Packs/FortiGate/ReleaseNotes/2_0_4.md

```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### Fortinet FortiGate Modeling Rule
+
+- Fixed an issue which caused the fields `xdm.source.host.ipv6_addresses` and `xdm.network.rule` to be populated with an empty string instead of a *null* value on cases there was no target value to be mapped to them.
```

Comment:
```suggestion
Fixed an issue which caused the fields `xdm.source.host.ipv6_addresses` and `xdm.network.rule` to be populated with an empty string instead of a *null* value in cases where there was no target value to be mapped to them.
```